packages/fluentui/docs/src/examples/components/Card/Slots/CardExamplePreview.tsx

```diff
@@ -0,0 +1,13 @@
+import { Card, Image } from '@fluentui/react';
+import * as React from 'react';
+
+const SimpleCard = () => (
+  <div style={{ width: '300px', border: '1px solid grey' }}>
+    <Card compact>
+      <Card.Preview id="cardMedia" fitted>
```

Comment:
Can we remove the `id`s props from all `Card.Preview` components or is there a reason for why those are added?

Reply:
Good catch, it was left from `Flex`-based draft